cardano-prometheus-tracker/cardano-prometheus-tracker.cabal

```diff
@@ -0,0 +1,56 @@
+cabal-version:      3.0
+name:               cardano-prometheus-tracker
+version:            0.1.0.0
+
+license:            Unlicense
+license-file:       LICENSE
+author:             Michael Karg
+maintainer:         mgoremeier@gmail.com
+
+category:           Web
+build-type:         Simple
+extra-doc-files:    CHANGELOG.md
+
```

Comment:
Add `copyright:              2019-2023 Input Output Global Inc (IOG), 2023-2025 Intersect.`
(also plz to `bench-data-publish`